SU2_CFD/src/output/CFlowCompOutput.cpp

```diff
@@ -414,6 +414,11 @@ void CFlowCompOutput::SetVolumeOutputFields(CConfig *config){
     AddVolumeOutput("Q_CRITERION", "Q_Criterion", "VORTEX_IDENTIFICATION", "Value of the Q-Criterion");
   }
 
+  // Mesh quality metrics, computed in CPhysicalGeometry::ComputeMeshQualityStatistics.
+  AddVolumeOutput("ORTHOGONALITY", "Orthogonality", "MESH_QUALITY", "Orthogonality Angle (deg.)");
+  AddVolumeOutput("ASPECT_RATIO",  "Aspect_Ratio",  "MESH_QUALITY", "CV Face Area Aspect Ratio");
+  AddVolumeOutput("VOLUME_RATIO",  "Volume_Ratio",  "MESH_QUALITY", "CV Sub-Volume Ratio");
+
```

Comment:
Seems all logical to me, is there a common place to share these options across the different outputs?

Reply:
Not that I am aware of. Even `COORD-X` etc is specifically mentioned in every output class.

Reply:
Ok, I guess we can leave that for the modules @talbring wants to implement.